book/bread-types/bread-types.tex

```diff
@@ -402,5 +402,5 @@ \section{Free standing bread}
 But after baking you will be rewarded with a beautiful bread
 with great taste and consistency.
 
-There is a full, dedicated recipe and tutorial
-for this type of bread in the~''\nameref{chapter:wheat-sourdough}''~Chapter.
+There is a dedicated recipe and tutorial for this type of bread in the
+``\nameref{chapter:wheat-sourdough}'' chapter.
```

Comment:
Probably the first unbreakable space was too much, not sure the 2nd shall go though. 

I would need to check but I think the quotes shall go.

Reply:
Please try it out on your end, as well.For me, removing any of the non-breaking spaces didn't help so I changed the sentence ever so slightly 😄